data/json/items/gunmod/magazine.json

```diff
@@ -12,13 +12,14 @@
     "symbol": ":",
     "color": "green",
     "location": "magazine",
-    "mod_targets": [ "rifle" ],
-    "acceptable_ammo": [ "223", "308", "50" ],
+    "mod_targets": [ "rifle", "shotgun" ],
+    "acceptable_ammo": [ "223", "308", "50", "shot" ],
```

Comment:
Please drop `shot`. Craftable guns specifically intended for belts is acceptable, converting factory firearms to snowflake-tier weapons isn't